packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -1715,8 +1752,11 @@ interface LinuxBuildImageProps {
 }
 
 // Keep around to resolve a circular dependency until removing deprecated ARM image constants from LinuxBuildImage
-// eslint-disable-next-line no-duplicate-imports, import/order
+/* eslint-disable no-duplicate-imports, import/order */
 import { LinuxArmBuildImage } from './linux-arm-build-image';
+import { LinuxArmLambdaBuildImage } from './linux-arm-lambda-build-image';
+import { LinuxLambdaBuildImage } from './linux-lambda-build-image';
+/* eslint-enable no-duplicate-imports, import/order */
```

Comment:
Since `LinuxArmLambdaBuildImage` and `LinuxLambdaBuildImage` reference `ComputeType`, it's necessary to place the import statements below `ComputeType`. I've adjusted their placement just beneath `ComputeType`. How does that look?

However, these classes only use `ComputeType` when generating error messages. It's also possible to move the import statements to the top of the file by modifying the error messages that don't use `ComputeType`. This option might be more appropriate.

Reply:
Thanks for clarifying, let's just keep them together as you did in the original commit then. It's clearer.